src/librustc_mir/build/matches/mod.rs

```diff
@@ -106,7 +105,7 @@ impl<'a,'tcx> Builder<'a,'tcx> {
 
     pub fn expr_into_pattern(&mut self,
                              mut block: BasicBlock,
-                             var_extent: CodeExtent, // lifetime of vars
+                             var_scope_id: ScopeId, // lifetime of vars
```

Comment:
This technically isn’t lifetime of the variable anymore, so the comment should be adjusted.